src/geo/transform.js

```diff
@@ -248,7 +246,7 @@ class Transform {
             this.pointCoordinate(new Point(this.width, this.height), z),
             this.pointCoordinate(new Point(0, this.height), z)
         ];
-        return tileCover(z, cornerCoords, options.reparseOverscaled ? actualZ : z, this._renderWorldCopies)
```

Comment:
Does this work for raster sources past their max zoom level? I think the previous behaviour was that it used the existing tiles from the max zoom level. Does this end up creating new tiles?